src/libraries/System.Private.CoreLib/src/System/IO/FileStatus.SetTimes.OtherUnix.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+namespace System.IO
+{
+    internal partial struct FileStatus
+    {
+        internal void SetCreationTime(string path, DateTimeOffset time) =>
+            SetLastWriteTime(path, time);
+
+        private void SetAccessOrWriteTime(string path, DateTimeOffset time, bool isAccessTime) =>
+            SetAccessOrWriteTimeCore(path, time, isAccessTime, checkCreationTime: false);
+
+        // This is not used on these platforms, but is needed for source compat
+        private Interop.Error SetCreationTimeCore(string path, DateTimeOffset time) =>
+            default(Interop.Error);
```

Comment:
`Interop.Error.ENOTSUP` is more appropriate (though it doesn't make a functional difference).

Reply:
I just put `default` there because I feel it emphasises how it isn't used (since it's not a real value). I can change it if that is better.

Reply:
@Jozkee should I change this or not? I think it's clearer how it currently is.

Reply:
I would suggest the following instead: 
```suggestion
            throw new InvalidOperationException();
```

Reply:
I will put this in the next commit.